packages/client/src/pages/ProfilePage/style.ts

```diff
@@ -0,0 +1,79 @@
+import Button from 'components/Button';
+import Form from 'components/Form';
+import styled, { css, CSSProp } from 'styled-components';
+
+import { AvatarType } from 'types';
+import defaultAvatar from 'assets/icons/default-avatar.svg';
+
+export const StyledForm = styled(Form)`
+  width: 928px;
+  padding: 40px 260px 60px;
+`;
+
+export const StAvatar = styled.div<AvatarType>`
+  position: relative;
+  width: 130px;
+  height: 130px;
+  border-radius: 50%;
+  margin: 0 auto;
+  background: ${({ image }) =>
+    `url(${image || defaultAvatar}) center no-repeat`};
+  background-color: ${props => props?.theme.COLOR_BACKGROUND_SECONDARY};
+  background-size: ${({ image }) => (image ? 'contain' : 'auto')};
+
+  &::before {
+    content: 'Поменять аватар';
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 110px;
+    height: 130px;
+    border-radius: 50%;
+    background-color: ${props => props?.theme.COLOR_ELEMENT_SECONDARY};
+    color: ${props => props?.theme.COLOR_TEXT_PRIMARY};
+    font-size: 14px;
+    line-height: 16px;
+    padding: 0 10px;
+
```

Comment:
![image](https://user-images.githubusercontent.com/54811697/227722902-5ce6c828-de87-4536-ad5c-42a216e0e5de.png)
```suggestion
   opacity: 0.8
```
?

Reply:
Мне нравится, добавила опасити

Reply:
еще стоит добавить
```suggestion
   cursor: pointer;
```
ещё нужно добавить курсор активный

Reply:
добавила